### DISCUSSION TITLE
Idiomatic way to search and replace similar to `:%s/from/to/gc`

### DISCUSSION THREAD

**Discussion by getreu** [helix-editor/helix#5912] [Q&A] [answered]
What is the Helix way to do Vim's:

     :%s/from/to/gc 

I experimented with `%` `s` _from_ `(` `)` `Alt-,` `c` _to_

[Writing down the above, I just realize how similar the keystrokes are!]
Are there other possibilities?

**Comment by Brixy** [comment]
- Please press `%s`.
- Type your search term.
- Press `Enter`.
- Press `c` and type your replacement.
- Press `ESC,` to get back to single selection.

**Reply by getreu** [reply]
Thank you @Brixy,

what you describe is Vim's `:%s/from/to/g`. 
Please note the `c` at the end in my example above: `:%s/from/to/gc`

From Vim's documentation:

> [c]     Confirm each substitution.  Vim highlights the matching string (with |hl-IncSearch|)

**Comment by Brixy** [comment, marked as answer]
This is probably not possible with multiple cursors.

For such cases I do a simple search (`/`), replace the first match (`c`), go to the next match (`n`) and press `.` to repeat the replacement.

This is probably no more work than using Vim’s `c` flag because you have to confirm each replacement in both cases.

I am also curious if there are better methods ;-)

**Reply by getreu** [reply]
Your workflow seem better than mine. I will try. Thank you.
